cirq-aqt/cirq_aqt/_version_test.py

```diff
@@ -3,4 +3,4 @@
 
 
 def test_version():
-    assert cirq_aqt.__version__ == "0.16.0.dev"
+    assert cirq_aqt.__version__ == "1.1.0.dev"
```

Comment:
Out of scope for this PR, but shouldn't we have version string in one place (and use others for overrides)?